mono/mini/interp/interp.c

```diff
@@ -735,15 +734,29 @@ static InterpMethodArguments* build_args_from_sig (MonoMethodSignature *sig, Mon
 		case MONO_TYPE_CLASS:
```

Comment:
if I understood correctly, one advantage of the current approach is that we don't need to all possible signatures beforehand.  but... it doesn't really matter, because we already went this route for the other way around. will look into it later.